src/System.DirectoryServices.AccountManagement/ref/System.DirectoryServices.AccountManagement.csproj

```diff
@@ -3,11 +3,14 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{404455B6-466C-4F78-9DCC-C37DCC0B75DA}</ProjectGuid>
+    <!-- UAPvNext is not yet mapped to netstandard2.0, manually duplicate this ref -->
+    <PackageTargetFramework>netstandard2.0;$(UAPvNextTFM)</PackageTargetFramework>
```

Comment:
OK I looked up the nuget change https://github.com/NuGet/NuGet.Client/pull/1600, so this should no longer be needed in our refs, as uap maps to netstandard2.0

Reply:
Are we already using this NuGet.Client version in corefx? Because I talked with @joperezr like 10 mins ago offline and he said that we still need them.